test/e2e/01-install.sh

```diff
@@ -66,13 +68,17 @@ set -e
 kubectl create secret tls -n tekton-pipelines tekton-results-tls --cert="${SSL_CERT_PATH}/tekton-results-cert.pem" --key="${SSL_CERT_PATH}/tekton-results-key.pem" || true
 
 echo "Installing Tekton Results..."
-kubectl kustomize "${ROOT}/test/e2e/kustomize" | ko apply -f -
+if [[ "$(uname -a)" =~ "ARM64" && "$(uname -m)" =~ "x86_64" ]]; then
```

Comment:
As suggested that check could go away, but for the record when using the `=~` operator, the right-hand side should be without quotes. See an example:
```
[ ~] -> [[ "abc" =~ "ab." ]] && echo "true"
[ ~] -> [[ "abc" =~ ab. ]] && echo "true"
true
```
The problem is that in quotes `.` loses it's special meaning to match a single character and instead is treated as `.` character part of the string.